tests/Bot.Builder.Community.Adapters.Alexa.Tests/ValidationHelperTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Alexa.NET.Request;
+using Bot.Builder.Community.Adapters.Alexa.Core;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Bot.Builder.Community.Adapters.Alexa.Tests
+{
+    public class ValidationHelperTests
+    {
+        [Fact]
+        public async Task ValidateRequestWithEmptySignatureUrlsShouldReturnFalse()
+        {
+            Microsoft.Extensions.Primitives.StringValues signatureChainUrls = string.Empty;
+            Microsoft.Extensions.Primitives.StringValues signatureHeaders = "test-signature-header";

```

Comment:
here shouldnt we be `using StringValues` directly?

Reply:
You're right. Applied, Thanks!